extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -384,6 +384,9 @@ class AmpAccordion extends AMP.BaseElement {
           () => {
             setStyles(section, {
               'overflow': 'hidden',
+              // Set the height of the section immediately, so it is correct
+              // until the animation below starts (on the next frame).
+              'height': `${headerHeight}px`,
```

Comment:
`px(headerHeight)` for consistency ?

Reply:
That would be the `px` function from `style`, which I would need to rename since the file already imports `px` from `transition`. I don't find it to be any more readable than just using a template string.